mantidproject/mantid
# Add category to new engineering diffraction interface doc page

**Description of work.**

Add a category to the engineering diffraction documentation page (for the new interface) so it shows up in the index page. 

Also removed the documentation page for the old C++ interface which has now been retired. This involved editing a reference in one of the old (3.7.1) release notes to remove the link and replace with plain text.

**To test:**

Build docs and check index page is updated (see screenshot on the linked issue) plus page is in following location:

build\docs\qthelp\interfaces\index.html

Fixes #29653.

*This does not require release notes* because **just adjusts the interfaces index page in the documentation**



---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
